src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -3554,6 +3554,7 @@ class spell_gen_stand : public SpellScript
 
         target->SetStandState(UNIT_STAND_STATE_STAND);
         target->HandleEmoteCommand(EMOTE_STATE_NONE);
+        target->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
```

Comment:
i don't think this should be here, the spell is also used in some other scripts (quests)